### ISSUE TITLE
List module takes posts away from blog.

### ISSUE THREAD

**Issue by PolCPP** [OrchardCMS/Orchard#7827]
I have an Orchard build that uses the Blog module with a pair of different blogs. 

We wanted to create a simple list of posts to populate a rss feed, so we created a List using the list module. The problem is, once you add a post to the list it gets detached to the post, and there is no way to re-add it back.

So in short, if a post is inside of a blog it cannot be used by List contentitems because it gets unlinked from the blog

**Comment by Hazzamanic** [comment]
Under the hood they both use the container property from commonpart to link the Blog/list with an item. To relink them you'll have to go into the db, it should be in a table called Common_CommonPartRecord, in the Container_id column, change this to the id of your blog.

If you want a new rss feed try using projections to query for the items you want. If you need specific ids Lombiq have a module where you can match content items by id, in this module https://github.com/Lombiq/Helpful-Extensions

There should probably be a warning when adding an item to a list if it is already within a container

**Comment by thekaveman** [comment]
An alternative to using Projections that gives you more fine-grained control (more similar to Lists), is to create a content type for configuring your RSS feed, with a ContentPicker Field setup to pick Blog Posts. 

This leaves the Blog Post contained within the Blog, while creating a distinct collection that can be queried and used elsewhere.

**Comment by sebastienros** [comment]
Correct, a list will be the new parent of the blog post. You can also use Taxonomies to tag blog posts to be aggregated in specific containers. Not sure a term exposes an rss feed though.
